apache-httpclient4/src/test/java/com/baeldung/httpclient/HttpClientHeaderV4LiveTest.java

```diff
@@ -1,15 +1,23 @@
 package com.baeldung.httpclient;
 
+import static org.apache.http.HttpHeaders.USER_AGENT;
```

Comment:
revert this change, we will not be adding the v4.5 example

Reply:
done